source/exec_cmd.c

```diff
@@ -119,6 +119,40 @@ static void updLocalPatches(const char* upd_path)
 	free(patch_ver);
 }
 
+static void backupCheats(const char* dst_path)
+{
+	char zip_path[256];
+	struct tm t;
+
+	// build file path
+	t = *gmtime(&(time_t){time(NULL)});
+	snprintf(zip_path, sizeof(zip_path), "%sGH-cheats_%d-%02d-%02d_%02d%02d%02d.zip", dst_path, t.tm_year+1900, t.tm_mon+1, t.tm_mday, t.tm_hour, t.tm_min, t.tm_sec);
+	if (!zip_directory(GOLDCHEATS_DATA_PATH, GOLDCHEATS_DATA_PATH, dst_path))
```

Comment:
Wrong path. should be `zip_path`.
```suggestion
	if (!zip_directory(GOLDCHEATS_DATA_PATH, GOLDCHEATS_DATA_PATH, zip_path))
```

Reply:
Also needing a dialog showing compression progress, currently it just freezes until zip is finished.